### PR TITLE
Add Apps card to the Dashboard

### PR DESCRIPTION
**Pull request by keoshi**
screen shot 2017-03-29 at 13 08 28

screen shot 2017-03-29 at 13 08 50

#### Proposed changes

This adds an WordPress Apps card to the bottom of the Dashboard page (At a Glance), which replaces the dedicated Apps page.

It also removes the original Apps page, and all references to it on common files.

Finally, it adds a new code — `target: 'apps-card’, button: ‘apps-download’` — so we can keep track to jumps to the WP Apps site.

#### How to test

* Login to your WP site
* Visit the Dashboard > At a Glance

Reference: p6TEKc-17n-p2

### CODE REVIEW

**Comment by eliorivero** [comment]
- all references to Apps must be removed from `NonAdminView` component

- The old SCSS file reference must be removed here
https://github.com/Automattic/jetpack/pull/6804/files#diff-45ad71ac8217a585f68ce2a9c7e80a10R53

- GUI tests are failing and need to be fixed To see them, run `yarn run test-gui` in the plugin folder. Let me know if you need a hand with this.

**Comment by eliorivero** [comment]
Also, from what I see in the screenshot, the space between the support card and the apps card is larger than between the other cards. I assume that's intentional?

**Comment by keoshi** [comment]
@eliorivero thank you so much for reviewing so quickly.

1 - Removes all Apps references from `NonAdminView`.
2 - Removed the original Apps styles import on main SCSS file.
3 - Ran `test-gui` command and can see the errors, but it seems like the tests themselves need to be fixed, not the code? If so, would appreciate your help to do that?
4 - The spacing is intentional indeed, to replicate the same spacing on the support card above.

Thanks again!

File: _inc/client/components/non-admin-view/index.jsx [hunk 1]
```diff
@@ -31,15 +30,7 @@ const NonAdminView = React.createClass( {
...
-				if ( this.props.userCanViewStats || this.props.isModuleActivated( 'protect' ) ) {
-					pageComponent = <AtAGlance { ...this.props } />;
-				} else {
-					// If routing took us to Dashboard but user can't view anything, fallback to Apps
-					pageComponent = <Apps { ...this.props } />;
-				}
-				break;
-			case '/apps':
-				pageComponent = <Apps { ...this.props } />;
+				pageComponent = <AtAGlance { ...this.props } />;
```
Comment (eliorivero) [inline _inc/client/components/non-admin-view/index.jsx]: What happens now when non admins can't see Stats and Protect isn't activated?

Comment (keoshi) [inline _inc/client/components/non-admin-view/index.jsx]: They see the simplified At a Glance screen, which contains the new Apps card, so the experience is identical.

**Comment by eliorivero** [comment]
Cool, thanks for solving these so fast!
Sometimes when the UI components are changed the tests need to be adapted. I'll do this and will commit to this branch.

**Comment by keoshi** [comment]
@eliorivero Thank _you_! I'll keep an eye on it, and see if the tests pass once those changes are pushed.

**Comment by eliorivero** [comment]
Pushed the tests, should be ok now.

**Comment by eliorivero** [comment]
It's building and passing tests now.

I checked the height and it's a bit different:

captura de pantalla 2017-03-29 a las 11 01 01

Before support block, the spacing is 64px and between support and apps card is 54px. Maybe because the last dash items are adding up margin

Adds 16px
captura de pantalla 2017-03-29 a las 11 02 37

Adds 48px
captura de pantalla 2017-03-29 a las 11 02 44

**Comment by keoshi** [comment]
@eliorivero Indeed, I must have forgotten to push the changes I made on the inspector. Thanks for double-checking that, just added that fix in the latest commit.

**Comment by eliorivero** [comment]
I've rebased this branch to solve conflicts after merging #6844.

**Comment by dereksmart** [comment]
Before merging, I'd like to see some way to dismiss this.  

Suggestion from @rickybanister in slack: 

> we were thinking possibly of a subtle dismiss action like an ‘I already use the app’ link

**Comment by jeffgolenski** [comment]
@dereksmart  Yeah buddddy. Can we add an italic link underneath that download button that says "I already use the app." as rick and you mention above?

**Comment by rickybanister** [comment]
👍

**Comment by rickybanister** [comment]
This is still listed as 'needs designer'— @dereksmart do you need an explicit mockup from us or are you good to go? I'd love for this to get moved in the next release.

**Comment by dereksmart** [comment]
@eliorivero I made the card dismissible.  If it looks good to you, please merge.  It looks like this 
screen shot 2017-04-24 at 8 33 24 pm

**Comment by eliorivero** [comment]
captura de pantalla 2017-04-24 a las 23 53 33

Updated to italicize only the link and not the button label. Updated to use sentence case in the button label since that's the writing style we're going for.

**Comment by rickybanister** [comment]
Nice work

**Pull request merged**